### ISSUE TITLE
[feature] Execute collect static only when versions change

### ISSUE THREAD

**Issue by nemesisdesign** [openwisp/docker-openwisp#246]
I have an idea of how we could deal with extreme slowness of calling collectstatic when using object storage:

- We could hash the output of `pip freeze` in the dashboard container
- Before calling `collectstatic` we check in redis to see if the current pip freeze hash is different from the one  saved there
- If it's different we call collectstatic and then save the new hash in redis

@pandafy @atb00ker what do you think?

**Comment by atb00ker** [comment]
I think it would work! 😄

**Comment by pandafy** [comment]
I agree, this will surely reduce the time for bringing up new container.
